[2025-06-02T18:27:37.123+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-04-13T00:00:00+00:00 [queued]>
[2025-06-02T18:27:37.159+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_docker_dag.run_test_task scheduled__2023-04-13T00:00:00+00:00 [queued]>
[2025-06-02T18:27:37.163+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-02T18:27:37.465+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_test_task> on 2023-04-13 00:00:00+00:00
[2025-06-02T18:27:37.486+0000] {standard_task_runner.py:60} INFO - Started process 513 to run task
[2025-06-02T18:27:37.577+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_docker_dag', 'run_test_task', 'scheduled__2023-04-13T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp3y2z5s_g']
[2025-06-02T18:27:37.626+0000] {standard_task_runner.py:88} INFO - Job 118: Subtask run_test_task
[2025-06-02T18:27:37.962+0000] {task_command.py:423} INFO - Running <TaskInstance: test_docker_dag.run_test_task scheduled__2023-04-13T00:00:00+00:00 [running]> on host c22acf1bbdfe
[2025-06-02T18:27:38.335+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_docker_dag' AIRFLOW_CTX_TASK_ID='run_test_task' AIRFLOW_CTX_EXECUTION_DATE='2023-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-13T00:00:00+00:00'
[2025-06-02T18:27:38.347+0000] {logging_mixin.py:188} INFO - âœ… Airflow DAG ran successfully inside Docker! After updating
[2025-06-02T18:27:38.350+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-02T18:27:38.402+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_docker_dag, task_id=run_test_task, execution_date=20230413T000000, start_date=20250602T182737, end_date=20250602T182738
[2025-06-02T18:27:38.550+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-02T18:27:38.678+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
